omxplayer/player.py

```diff
@@ -205,6 +210,14 @@ def _setup_omxplayer_process(self, source):
         atexit.register(self.quit)
         return process
 
+    def _terminate_process(self, process):
+        try:
+            process_group_id = os.getpgid(process.pid)
+            os.killpg(process_group_id, signal.SIGTERM)
+            logger.debug('SIGTERM Sent to pid: %s' % process_group_id)
+        except OSError:
+            logger.error('Could not find the process to kill')
```

Comment:
Under what circumstances might we get an OSError? If `process_group_id` no longer exists by the time `os.killpg` occurs?

Reply:
I think I've seen exactly that, yes. Also, this is just code that was already there, I just moved it. :-)